pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -56,6 +58,10 @@ func ProvideService(cfg *setting.Cfg, sqlStore db.DB, ac ac.AccessControl,
 	if features.IsEnabledGlobally(featuremgmt.FlagSsoSettingsApi) {
 		ssoSettingsApi := api.ProvideApi(svc, routeRegister, ac)
 		ssoSettingsApi.RegisterAPIEndpoints()
+
+		if isGrafanaHA(cfg) {
```

Comment:
We probably always need to schedule the reload functionality. This is a flag to get _live_ working correctly in a HA setup but there is nothing stopping someone from running HA without this flag set. Also _live_ can be disabled

Reply:
Thanks for confirming, I wasn't sure about that flag either.